libreoffice-core/officecfg/registry/data/org/openoffice/Office/Common.xcu

```diff
@@ -524,6 +524,128 @@
         <value oor:external="com.sun.star.configuration.backend.DesktopBackend SourceViewFontHeight"/>
       </prop>
     </node>
+    <node oor:name="Substitution">
```

Comment:
I think moving forward we should also do something similar to what I mentioned above about comments in C++ with XML changes.

```suggestion
    <!-- MACRO-1518: Fix Calibri and Cambria display { -->
    <node oor:name="Substitution">
```